frontend/src/store/actions/auth.js

```diff
@@ -0,0 +1,49 @@
+import * as safeStorage from '../../utils/safe_storage';
+import { fetchUserDetails } from '../../network/auth';
+
+
+export const types = {
+  SET_USER_DETAILS: 'SET_USER_DETAILS',
+  SET_TOKEN: 'SET_TOKEN',
+  CLEAR_SESSION: 'CLEAR_SESSION'
+};
+
+export function clearUserDetails() {
+  return {
+    type: types.CLEAR_SESSION,
+  };
+}
+
+export const logout = () => dispatch => {
```

Comment:
For Firefox, I wasn't able to replicate the problem Wille described where these come back after a refresh in same tab after logging out. It cleared them out completely.

Reply:
Thanks, @thadk !